Mathlib/Tactic/Linarith/Verification.lean

```diff
@@ -202,7 +202,7 @@ def proveFalseByLinarith (cfg : LinarithConfig) : MVarId → List Expr → MetaM
       let (comps, max_var) ← linearFormsAndMaxVar cfg.transparency inputs
       trace[linarith.detail] "... finished `linearFormsAndMaxVar`."
       trace[linarith.detail] "{comps}"
-      let oracle := cfg.oracle.getD FourierMotzkin.produceCertificate
+      let oracle := cfg.oracle.getD SimplexAlgorithm.produceCertificate
```

Comment:
```suggestion
      -- If needed, we can switch back to the Fourier-Motzkin generator using
      -- let oracle := cfg.oracle.getD FourierMotzkin.produceCertificate
      let oracle := cfg.oracle.getD SimplexAlgorithm.produceCertificate
```

Reply:
I don't think we need this; we can already switch back with `linarith (config := { oracle := some .fourierMotzkin})` (after the changes I just pushed to fix dot notation.